docs/scripting/functions/db_close.md

```diff
@@ -1,58 +1,73 @@
 ---
 id: db_close
 title: db_close
-description: Closes an SQLite database that was opened with db_open.
-tags: [sqlite]
+description: Closes a SQLite database connection that was opened with `db_open`.
+keywords:
+    - sqlite
 ---
 
 :::warning
 
-This function starts with lowercase letter.
+This function starts with a lowercase letter.
 
 :::
 
 ## Description
 
-Closes an SQLite database that was opened with db_open.
+Closes a SQLite database connection that was opened with [db_open](db_open).
 
-| Name  | Description                                                           |
-| ----- | --------------------------------------------------------------------- |
-| DB:db | The handle of the database connection to close (returned by db_open). |
+| Name  | Description                                                                      |
+| ----- | -------------------------------------------------------------------------------- |
+| DB:db | The handle of the database connection to close (returned by [db_open](db_open)). |
 
 ## Returns
 
-1: The function executed successfully.
+1: This function executed successfully.
 
-0: The function failed to execute. May mean that the database handle specified is not open.
+0: This function failed to execute. This could mean that the database connection handle is invalid.
 
 ## Examples
 
 ```c
-new DB:db_handle;
+static DB:g_DBConnectionHandle;
```

Comment:
`g_` does not currently match our style guide.

Reply:
My bad, I remembered the prefix `g_`, not `g`. To be fair personally I am not a fan prefixing global variables at all.